app/submitter/converter.py

```diff
@@ -192,7 +193,7 @@ def _encode_value(value):
     if isinstance(value, int):
         return str(value)
     elif isinstance(value, datetime):
-        return value.strftime(settings.SDX_DATE_FORMAT)
+        return value.strftime(SDX_DATE_FORMAT)
```

Comment:
We possibly should have a test. Do we know what format we are actually sending dates as?